### DISCUSSION TITLE
Feature Request: Allow to change sender name

### DISCUSSION THREAD

**Discussion by Fovty** [Cloud9Developer/Jellyfin-Newsletter-Plugin#28] [Ideas]
It would be nice to have an input field besides the "From Address" to change the sender name too.

**Comment by Cloud9Developer** [comment]
I'm not sure what you mean exactly? Could you provide an example?

**Reply by Fovty** [reply]
Never mind, I just found out that under the "From Address" you can also enter a display name in this format: "Jellyfin \<newsletter@domain.com\>". I was used to have one field for the sender name and one for the sender address.
